src/BankConsole/BankClient.cs

```diff
@@ -79,12 +79,32 @@ private void quit()
 
         private void newAccount()
         {
-            Console.Write("Specify the origin of the account (Local, Rural, Foreign): ");
+            Console.Write("Specify the 'origin' of the account (Local, Rural, Foreign): ");
             string accountOrigin = Console.ReadLine();
-            currentAccount = bankService.newAccount(accountOrigin);
+            Console.Write("Specify the 'type' of the account\n\t1 (Checking account)\n\t2 (Saving account)\n:");
+            string accountType = TypeAccountName(Console.ReadLine());
+            currentAccount = bankService.NewAccount(accountType, accountOrigin);
             Console.WriteLine("Your new account number is: " + currentAccount);
         }
 
+        private string TypeAccountName(string input)
+        {
+            string typeName;
```

Comment:
I would like to see a Swtich Expression here

Reply:
Thanks you for the review, I didn't notice that, but I will do